temporal-sdk/src/main/java/io/temporal/common/interceptors/WorkflowOutboundCallsInterceptor.java

```diff
@@ -483,7 +478,15 @@ public DynamicSignalHandler getHandler() {
 
   void upsertSearchAttributes(Map<String, Object> searchAttributes);
 
-  Object newThread(Runnable runnable, boolean detached, String name);
+  /**
+   * Intercepts creation of the workflow child thread
```

Comment:
I fear that can get confused with child workflows. Maybe add to the comment that it is not related to child workflows in any way.